packages/app/obojobo-document-engine/src/scripts/oboeditor/components/objectives/objective-list-modal.js

```diff
@@ -0,0 +1,207 @@
+import './objective-list-modal.scss'
+
+import Common from 'obojobo-document-engine/src/scripts/common'
+import React from 'react'
+import uuid from 'obojobo-document-engine/src/scripts/common/util/uuid'
+import ObjectiveItem from './objective-item'
+import ObjectiveInput from './objective-input'
+import ObjectiveEdit from './objective-edit'
+
+const { SimpleDialog } = Common.components.modal
+const { Button } = Common.components
+
+class ObjectiveListModal extends React.Component {
+	constructor(props) {
+		super(props)
+		this.inputRef = React.createRef()
+		this.state = {
+			objectives: this.props.content.objectives ?? [],
+			globalObjectives: this.props.objectiveContext.objectives ?? []
+		}
+
+		this.state.newObjective = false
+
+		this.state.editMode = false
+		this.state.editData = null
+
+		this.addObjective = this.props.objectiveContext.addObjective
+		this.removeObjective = this.props.objectiveContext.removeObjective
+		this.updateObjective = this.props.objectiveContext.updateObjective
+
+		this.createNewObjective = this.createNewObjective.bind(this)
+		this.handleObjectiveInput = this.handleObjectiveInput.bind(this)
+
+		this.objectiveListHeader = (
+			<div
+				style={{
+					margin: '-0.7em 0em 1em 0em'
+				}}
+			>
```

Comment:
Add a `className` to this and handle the styling with a rule in the adjacent .scss file.